state_lattice_planner:
    map_frame_id : map
    lane_width : 1.3
    vehicle_width : 0.3
    sampling_number : 11
    distanceCost_gain : 1.0
    obstacleCost_gain : 3
    loop_rate : 20